cmd/main.go

```diff
@@ -293,3 +331,32 @@ func run(ctx context.Context, kubeClient kubernetes.Interface, er record.EventRe
 	// Hence, we must manually exit now that we know controllers have been shutdown properly.
 	os.Exit(0)
 }
+
+// dcosLogin using a service account secret
+func dcosLogin(saConfig dcos.ServiceAccountOptions) (*dcos.APIClient, error) {
+	// Empty config, without auth token
+	config := dcos.NewConfig(nil)
+	config.SetURL("https://leader.mesos")
```

Comment:
I remember to use `master.mesos` instead of this url. It was recommended by dcos devs however I don't exactly remember the reason but I believe it was due to (certain net issues) where each node was giving a different leader.

Reply:
Weird because I copied this from their code.

Reply:
I think we'll be OK with either, but `master.mesos` is what I've been told to use before too. Perhaps check with #core?

Reply:
Done.